[2025-04-20T14:38:53.573+0000] {processor.py:186} INFO - Started process (PID=165) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:38:53.574+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:38:53.581+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:38:53.578+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:38:53.598+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:38:53.590+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:38:53.602+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:38:53.634+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.075 seconds
[2025-04-20T14:39:24.613+0000] {processor.py:186} INFO - Started process (PID=2475) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:39:24.618+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:39:24.645+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:39:24.644+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:39:24.722+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:39:24.713+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:39:24.734+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:39:24.898+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.341 seconds
[2025-04-20T14:39:56.317+0000] {processor.py:186} INFO - Started process (PID=5124) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:39:56.344+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:39:56.366+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:39:56.366+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:39:56.408+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:39:56.392+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:39:56.410+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:39:56.524+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.237 seconds
[2025-04-20T14:40:28.114+0000] {processor.py:186} INFO - Started process (PID=5990) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:40:28.144+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:40:28.264+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:40:28.263+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:40:28.570+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:40:28.467+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:40:28.595+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:40:28.907+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 1.015 seconds
[2025-04-20T14:41:00.624+0000] {processor.py:186} INFO - Started process (PID=8008) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:41:00.648+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:41:00.675+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:41:00.665+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:41:00.735+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:41:00.717+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:41:00.747+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:41:01.441+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.905 seconds
[2025-04-20T14:41:33.308+0000] {processor.py:186} INFO - Started process (PID=8458) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:41:33.352+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:41:33.420+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:41:33.417+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:41:33.515+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:41:33.496+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:41:33.542+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:41:34.031+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.918 seconds
[2025-04-20T14:42:05.241+0000] {processor.py:186} INFO - Started process (PID=9739) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:42:05.261+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:42:05.322+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:42:05.302+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:42:05.454+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:42:05.402+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:42:05.463+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:42:06.005+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.934 seconds
[2025-04-20T14:42:38.951+0000] {processor.py:186} INFO - Started process (PID=11230) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:42:38.974+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:42:39.102+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:42:39.097+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:42:39.268+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:42:39.209+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:42:39.275+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:42:39.476+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.599 seconds
[2025-04-20T14:43:12.383+0000] {processor.py:186} INFO - Started process (PID=11729) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:43:12.408+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:43:12.430+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:43:12.427+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:43:12.631+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:43:12.574+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:43:12.637+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:43:12.831+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.543 seconds
[2025-04-20T14:43:44.238+0000] {processor.py:186} INFO - Started process (PID=12185) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:43:44.251+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:43:44.294+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:43:44.275+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:43:44.359+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:43:44.332+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:43:44.362+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:43:44.747+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.564 seconds
[2025-04-20T14:44:19.473+0000] {processor.py:186} INFO - Started process (PID=12614) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:44:19.514+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:44:19.564+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:44:19.541+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:44:19.880+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:44:19.803+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:44:19.890+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:44:20.620+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 2.524 seconds
[2025-04-20T14:44:56.401+0000] {processor.py:186} INFO - Started process (PID=12929) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:44:56.424+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:44:56.430+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:44:56.428+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:44:56.478+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:44:56.467+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:44:56.479+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:44:56.584+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.589 seconds
[2025-04-20T14:45:27.856+0000] {processor.py:186} INFO - Started process (PID=13540) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:45:27.866+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:45:27.878+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:45:27.876+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:45:27.982+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:45:27.956+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:45:27.991+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:45:28.449+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.712 seconds
[2025-04-20T14:46:17.943+0000] {processor.py:186} INFO - Started process (PID=15064) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:46:18.002+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:46:18.058+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:46:18.056+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:46:18.157+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:46:18.111+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:46:18.162+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:46:19.101+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 1.270 seconds
[2025-04-20T14:47:22.468+0000] {processor.py:186} INFO - Started process (PID=16515) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:47:22.477+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:47:22.485+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:47:22.484+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:47:22.571+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:47:22.561+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:47:22.580+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:47:22.861+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.432 seconds
[2025-04-20T14:47:53.966+0000] {processor.py:186} INFO - Started process (PID=17888) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:47:53.974+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:47:53.977+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:47:53.976+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:47:54.028+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:47:54.020+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:47:54.048+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:47:54.173+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.274 seconds
[2025-04-20T14:48:26.104+0000] {processor.py:186} INFO - Started process (PID=18414) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:48:26.185+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:48:26.251+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:48:26.232+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:48:26.578+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:48:26.497+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:48:26.585+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:48:27.534+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 1.543 seconds
[2025-04-20T14:48:58.876+0000] {processor.py:186} INFO - Started process (PID=19057) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:48:58.904+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:48:58.932+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:48:58.924+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:48:59.030+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:48:59.013+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:48:59.040+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:48:59.153+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.583 seconds
[2025-04-20T14:49:31.395+0000] {processor.py:186} INFO - Started process (PID=19501) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:49:31.476+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:49:31.568+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:49:31.539+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:49:32.062+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:49:31.941+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:49:32.064+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:49:32.491+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 1.600 seconds
[2025-04-20T14:50:05.804+0000] {processor.py:186} INFO - Started process (PID=20695) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:50:05.820+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:50:05.828+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:50:05.828+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:50:05.884+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:50:05.870+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:50:05.888+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:50:06.128+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.400 seconds
[2025-04-20T14:50:37.385+0000] {processor.py:186} INFO - Started process (PID=21741) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:50:37.459+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:50:37.477+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:50:37.476+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:50:37.666+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:50:37.636+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:50:37.674+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:50:38.201+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 1.144 seconds
[2025-04-20T14:51:09.490+0000] {processor.py:186} INFO - Started process (PID=22524) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:51:09.515+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:51:09.579+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:51:09.578+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:51:09.698+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:51:09.684+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:51:09.706+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:51:09.848+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.543 seconds
[2025-04-20T14:51:41.440+0000] {processor.py:186} INFO - Started process (PID=23276) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:51:41.477+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:51:41.492+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:51:41.488+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:51:41.583+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:51:41.562+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:51:41.607+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:51:42.154+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.876 seconds
[2025-04-20T14:52:14.064+0000] {processor.py:186} INFO - Started process (PID=24824) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:52:14.125+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:52:14.131+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:52:14.130+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:52:14.362+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:52:14.323+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:52:14.364+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:52:14.606+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.666 seconds
[2025-04-20T14:52:51.455+0000] {processor.py:186} INFO - Started process (PID=25658) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:52:51.551+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:52:51.618+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:52:51.581+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:52:51.738+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:52:51.698+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:52:51.739+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:52:51.958+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.710 seconds
[2025-04-20T14:53:23.380+0000] {processor.py:186} INFO - Started process (PID=26118) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:53:23.430+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:53:23.467+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:53:23.466+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:53:23.662+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:53:23.616+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:53:23.665+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:53:23.872+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.645 seconds
[2025-04-20T14:53:56.077+0000] {processor.py:186} INFO - Started process (PID=27250) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:53:56.099+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:53:56.135+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:53:56.103+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:53:56.261+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:53:56.248+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:53:56.265+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:53:56.414+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.469 seconds
[2025-04-20T14:54:50.974+0000] {processor.py:186} INFO - Started process (PID=29089) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:54:51.007+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:54:51.024+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:54:51.016+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:54:51.219+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:54:51.174+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:54:51.234+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:54:51.459+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.731 seconds
[2025-04-20T14:55:22.387+0000] {processor.py:186} INFO - Started process (PID=29490) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:55:22.396+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:55:22.424+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:55:22.417+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:55:22.524+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:55:22.515+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:55:22.531+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:55:22.790+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.455 seconds
[2025-04-20T14:55:53.743+0000] {processor.py:186} INFO - Started process (PID=30711) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:55:53.785+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:55:53.826+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:55:53.824+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:55:54.081+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:55:54.057+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:55:54.091+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:55:55.356+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 1.859 seconds
[2025-04-20T14:56:30.138+0000] {processor.py:186} INFO - Started process (PID=32170) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:56:30.149+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:56:30.164+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:56:30.155+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:56:30.299+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:56:30.270+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:56:30.304+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:56:30.434+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.367 seconds
[2025-04-20T14:57:02.033+0000] {processor.py:186} INFO - Started process (PID=32713) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:57:02.176+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:57:02.237+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:57:02.236+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:57:02.693+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:57:02.577+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:57:02.709+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:57:03.254+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 1.447 seconds
[2025-04-20T14:57:40.025+0000] {processor.py:186} INFO - Started process (PID=33818) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:57:40.075+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:57:40.177+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:57:40.104+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:57:40.411+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:57:40.387+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:57:40.418+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:57:43.384+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 4.211 seconds
[2025-04-20T14:58:16.240+0000] {processor.py:186} INFO - Started process (PID=34529) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:58:16.587+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:58:16.612+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:58:16.611+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:58:16.773+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:58:16.758+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:58:16.777+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:58:17.706+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 1.853 seconds
[2025-04-20T14:59:07.120+0000] {processor.py:186} INFO - Started process (PID=35145) to work on /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:59:07.129+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/hn_nhatot_scraping_dag.py for tasks to queue
[2025-04-20T14:59:07.171+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:59:07.159+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:59:07.315+0000] {logging_mixin.py:190} INFO - [2025-04-20T14:59:07.306+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/hn_nhatot_scraping_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/hn_nhatot_scraping_dag.py", line 13, in <module>
    from scraping.HNSaveLinkNhatot import scrape_links as save_links
ModuleNotFoundError: No module named 'scraping'
[2025-04-20T14:59:07.319+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/hn_nhatot_scraping_dag.py
[2025-04-20T14:59:07.513+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/hn_nhatot_scraping_dag.py took 0.524 seconds
